### PR TITLE
Added Gitter Icon

### LINKED ISSUE #11: Add gitter icon
As can be seen in the navbar and social links of the organization the icon for the gitter channel is not available backed up with the link to the room.
Please add it.

### PR DESCRIPTION
**Pull request by evi1haxor**
#### Type of Change:

- [x] Code
- [ ] Blog
- [ ] Resources

#### Description

Fixes #11 

CZ website now has a gitter icon. 

#### Screenshots

image

#### Other comments

Media queries are yet to be written so making a draft PR for now.

### CODE REVIEW

**Comment by czgdp1807** [comment]
Thanks for the addition. The screenshot LGTM. Will review it soon.

File: assets/css/main.scss [hunk 1]
```diff
@@ -69,3 +69,121 @@ $background_image: '{{site.background_image}}';
...
+		background-color: white;
+		width: 1.6px;
+		height: 11px;
+	}
+	.logo-holder {
+		position: relative;
+		width: 10px;
+		height: 26px;
+	}
+}
```
Comment (czgdp1807) [inline assets/css/main.scss]: Please add a  newline here.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+push.py
```
Comment (czgdp1807) [inline .gitignore]: Please don't modify the gitignore. Delete the file `push.py` as it is not of any use to the website.

Comment (evi1haxor) [reply .gitignore]: I'll keep it in mind next time.

**Comment by czgdp1807** [comment]
Is the work complete on this?

**Comment by evi1haxor** [comment]
I'm onto it. Was AFK since last two days.

**Comment by czgdp1807** [comment]
The radius of gitter icon's border isn't matching with the ones of other icons. Can something be done regarding this?
Screenshot from 2019-07-31 14-34-28

Please add the gitter icon here too.
Screenshot from 2019-07-31 14-34-48

Thanks.

**Comment by czgdp1807** [comment]
Please configure or editor for always appending a single new line at the end of new line.

**Comment by evi1haxor** [comment]
> The radius of gitter icon's border(on smaller screens) isn't matching with the ones of other icons. Can something be done regarding this?

I'll look into it. Actually I just estimated their properties cause i couldn't find fontawesome icon's css.

>Please configure your editor for always appending a single new line at the end of the file.

Sure.

> Please add the gitter icon here too.(in the footer, where other contact information and contact form is given)

Sure.

**Comment by czgdp1807** [comment]
The gitter icon is too bright. Probably you are using the same styling classes for the icon in the header and the footer. May be, some minor changes are required to modify the color in the footer.

Screenshot from 2019-08-04 12-46-43

**Comment by evi1haxor** [comment]
Yes. It's incomplete yet. Will finish it soon.

Can you please check whether the radius is doing good or not?

**Comment by czgdp1807** [comment]
> Can you please check whether the radius is doing good or not?

Yes, everything else is fine and looks okay. It will be in after you make changes to the color and the border of the gitter icon in the footer.

**Comment by evi1haxor** [comment]
I'm done on my side. Please review.

**Comment by czgdp1807** [comment]
Thanks.

**Pull request merged**